operations/patch_util.go

```diff
@@ -676,6 +681,24 @@ func loadGitData(dir, branch, ref, commits string, format bool, extraArgs ...str
 	}, nil
 }
 
+func gitGetRemote(dir, owner, repo string) (string, error) {
+	out, err := gitCmdWithDir("remote", dir, "-v")
+	if err != nil {
+		return "", errors.Wrap(err, "getting git remotes")
+	}
+
+	lines := strings.Split(out, "\n")
+	partial := strings.ToLower("github.com/" + owner + "/" + repo)
+	for _, line := range lines {
+		f := strings.Fields(line) // ["remote-name", "remote-url"]
```

Comment:
It may be helpful to add a comment above this to mention that the format of each line of `git remote -v` should be `<remote_name> <url> <fetch_or_pull>` to make the string parsing logic easier to reason through.